src/model/media-player.ts

```diff
@@ -76,7 +76,10 @@ export class MediaPlayer {
     if (this.members.length > 1 && this.config.adjustVolumeRelativeToMainPlayer) {
       return this.getAverageVolume();
     } else {
-      return 100 * this.volumePlayer.attributes.volume_level;
+      if (this.volumePlayer.attributes.volume_level)
+        return 100 * this.volumePlayer.attributes.volume_level;
+      else
+        return 0
```

Comment:
please add braces to if clause
```suggestion
      if (this.volumePlayer.attributes.volume_level) {
        return 100 * this.volumePlayer.attributes.volume_level;
      } else {
        return 0
      }
```

Reply:
I added the braces as requested, and pushed to my repo branch.